src/server/routes.js

```diff
@@ -44,6 +44,10 @@ export default (server, app) => {
   // Support older assets from website
   server.use('/public/images', express.static(path.join(__dirname, '../static/images')));
 
+  server.get('/manifest.json', maxAge(7200), corsEnabled(), (req, res) => {
```

Comment:
We can use the `cors` package for that and delete the middleware, see: https://expressjs.com/en/resources/middleware/cors.html

Reply:
done